fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -135,6 +139,15 @@ public boolean executeTaskRun() throws Exception {
             status.setErrorCode(errorCode);
             return false;
         }
+
+        // Execute post task action, but ignore any exception
+        if (StringUtils.isNotEmpty(taskRunContext.getPostRun())) {
```

Comment:
What is the performance of the analyze statement?

suggestion:
nit:To put it simply change, better add a switch to decide whether to execute postRun, or only use for MV.
If it is only used for MV, you can use task.getSource to determine the source of MV.

If it is complicated change, I think that this part may also be configurable. For example, if the user understands the details, he can configure whether to analyze full, analyze sample or not analyze after refreshing.

Reply:
- Without analyzing the table, the MV rewriter barely works
- It's good to add a switch to configure the analyze behavior, I would do it